frameworks/kafka/tests/test_tls.py

```diff
@@ -120,24 +122,33 @@ def test_tls_ciphers(kafka_service_tls):
         config.SERVICE_NAME,
         'describe',
         json=True), '').rstrip().split(','))
-    possible_ciphers = set(map(cipher_suites.rfc_name, sdk_security.openssl_ciphers()))
+
+    openssl_ciphers = sdk_security.openssl_ciphers()
+
+    missing_openssl_ciphers = sdk_security.missing_openssl_ciphers(openssl_ciphers)
+    log.warning("\n%s OpenSSL ciphers missing from the cipher_suites module:", len(missing_openssl_ciphers))
+    log.warning("\n".join(sdk_utils.sort(list(missing_openssl_ciphers))))
+
+    possible_ciphers = set(filter(partial(is_not, None),
```

Comment:
If `openssl_ciphers` is a `set` and `missing_openssl_ciphers` is also a `set`, then isn't 
```python
possible_ciphers = set(map(cipher_suites.rfc_name, openssl_ciphers - missing_openssh_ciphers))
```
?
One may also want to update the variable names to indicate the difference between RFC and SSL names.

Reply:
> possible_ciphers = set(map(cipher_suites.rfc_name, openssl_ciphers - missing_openssh_ciphers))

Ah yes, that's better.

Regarding naming I was prefixing `openssl` when they were OpenSSL names and not adding any prefix when they were RFC names. Do you think we should instead prefix `rfc`?

Reply:
https://github.com/mesosphere/dcos-commons/pull/2518/commits/7364f4d353b177e4be6f782a1ac1fb7d0fbb6221

Reply:
Hmmm ... yes, you're right. I missed the `openssl_`. That's fine then.